recipes/met/runner.sh

```diff
@@ -12,7 +12,7 @@ ACL=private
     rm -rf output && mkdir -p output
 
 
-    files=$(axway_ls -nrt Met_Museum/Attendance | grep .csv | awk '{print $NF}')
+    files=$(axway_ls -nrt Met_Museum_2/Attendance | grep .csv | awk '{print $NF}')
```

Comment:
Full folder name on Axway is "Met_Museum_2 (NYCAnalytics@cityhall.nyc.gov)"

Reply:
@YuliaZamriy there are two folders in axway. One with and one without the email address. Why are there two?

Reply:
@SteveScott That's a very good question. We should ask DOITT about that. But Met_Museum_2 doesn't have any data, while "Met_Museum_2 (NYCAnalytics@cityhall.nyc.gov)" has the most recent datasets.